rfcs/0001-bluemoji.md

```diff
@@ -35,10 +35,11 @@ order of priorities.
 - _Bluemoji Alias_, _alias_, _colon-wrapped alias_, _dotted-alias_: This is how
   the Bluemoji is named, reflected in the record's `rkey`. When detected by the
   Bluemoji Facet (see below), these are decorated with metadata allowing clients
-  to render the Bluemoji. The record name itself must match `\b[a-z0-9-]+\b` and
-  does not include `:` characters; when written prior to faceting, it is written
-  wrapped in `:` character (e.g., `:alias-name:`), hereafter referred to as a
-  "colon-wrapped alias" or simply "dotted-alias".
+  to render the Bluemoji. The record name itself must match
+  `^\b(?<!-)[a-z0-9-]+(?!<-)\b$` and does not include `:` characters; when
```

Comment:
Which RegExp syntax is this? In JS this should probably be `^(?!-)[a-z0-9-]+(?<!-)$` for negative lookahead and lookbehind `-`, respectively. `\b` isn't needed at the start or end of the string.